Tests/Resources/recrash-report.json

```diff
@@ -0,0 +1,818 @@
+{
+    "recrash_report": {
+        "report": {
+            "version": "3.2.0",
+            "id": "E3584241-76E0-4C07-9E46-77BA97E59404",
+            "process_name": "process",
+            "timestamp": 1593717988,
+            "type": "standard"
+        },
+        "binary_images": [
```

Comment:
H: Could you please trim down this event a bit? You might get around without the `binary_images` altogether or just keep a couple. Same goes for other lists like `crashed_thread`

Reply:
I already trimmed it down a lot 😁. It is a real report that I censored a bit. I thought it makes sense to have a realistic test case.

Reply:
It's still realistic without that many `binary_images`, 1 is enough

Reply:
We only have now 1 binary image.